options/locale/locale_en-US.ini

```
@@ -1293,7 +1293,10 @@ issues.filter_label = Label
 issues.filter_label_exclude = `Use <code>alt</code> + <code>click/enter</code> to exclude labels`
 issues.filter_label_no_select = All labels
 issues.filter_milestone = Milestone
-issues.filter_milestone_no_select = All milestones
+issues.filter_milestone_all = All milestones
+issues.filter_milestone_no_selected = No milestones
```

```suggestion
issues.filter_milestone_none = No milestones
```
Is this maybe more explicit when reading the key name?

---

I don't know but I followed your suggestion.